receiver/signalfxreceiver/receiver.go

```diff
@@ -218,7 +218,7 @@ func (r *sfxReceiver) writeResponse(ctx context.Context, resp http.ResponseWrite
 		return
 	}
 
-	resp.WriteHeader(http.StatusAccepted)
+	resp.WriteHeader(http.StatusOK)
```

Comment:
I remember there was some problem with this. I think SignalFx agent is expecteding 201.

Reply:
Clarified that the expected is StatusOk (200) + a message body equivalent with `"OK"` which is already the case.